rdkit/Chem/UnitTestMol2.py

```diff
@@ -0,0 +1,58 @@
+# $Id$
```

Comment:
This is an ok basic test, but UnitTestMol2.py needs to be added to the list of files to be tested in $RDBASE/rdkit/Chem/test_list.py.